requirements-test.txt

```diff
@@ -1,3 +1,4 @@
+backports.unittest-mock==1.3
```

Comment:
This produces a warning when you run the tests
```
=========================================== warnings summary ============================================
None
  Module already imported so can not be re-written: backports

-- Docs: http://doc.pytest.org/en/latest/warnings.html
```

Do you know how to get rid of this?

Reply:
I do not know anything about this, sorry. A cursory google search suggests that this thread might hold the key: https://github.com/pytest-dev/pytest-cov/issues/148#issuecomment-275799903

Reply:
Per discussion [here](https://github.com/pypa/setuptools/issues/1038) tried downgrading backports.unittest-mock to 1.2 and that seems to have fixed it for me. Not sure why. :-p